core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/kurtosis_base_builtin_internal.go

```diff
@@ -61,3 +64,38 @@ func (builtin *KurtosisBaseBuiltinInternal) GetArguments() *builtin_argument.Arg
 func (builtin *KurtosisBaseBuiltinInternal) GetPosition() *KurtosisBuiltinPosition {
 	return builtin.position
 }
+
+func printWarningForArguments(arguments []*builtin_argument.BuiltinArgument, builtin *KurtosisBaseBuiltin) {
+	// if instruction is deprecated, print the deprecated warning for the instruction
+	// ignore the warnings associated with arguments
+	if builtin.Deprecation != nil {
+		warningMessage := getFormattedWarningMessageForInstruction(builtin.Deprecation, builtin.Name)
+		starlark_warning.PrintOnceAtTheEndOfExecutionf("%v %v", starlark_warning.WarningConstant, warningMessage)
```

Comment:
TIL about [`%v`](https://golang.google.cn/pkg/fmt/#:~:text=The%20default%20format%20for%20%25v%20is%3A).